spec-main/api-protocol-spec.ts

```diff
@@ -919,8 +919,8 @@ describe('protocol module', () => {
       await protocol.unregisterProtocol('stream');
     });
 
-    it('does not successfully play videos with stream: false on streaming protocols', async () => {
-      await streamsResponses(standardScheme, 'error');
+    it('successfully plays videos with stream: false on streaming protocols', async () => {
+      await streamsResponses(standardScheme, 'play');
```

Comment:
Seems like this fixes the root cause for https://github.com/electron/electron/pull/22955 and the stream option seems unnecessary now ?

/cc @pfrazee

Reply:
I'll need to look into this - the `stream` flag tells the video/audio player to use stream behaviors instead of fully-buffered behaviors, so what might have happened is this PR fixed the fully-buffered behavior but it still isn't streaming, which would mean playback requires a full download.

Reply:
Thanks! that clarifies my question. Then the test case should be modified to rather denote streaming content when `stream: true` vs completely buffered content download when `stream: false`.

The current change to the test case doesn't differentiate this fix from https://github.com/electron/electron/pull/22955

Reply:
I've updated the test names to indicate the difference, is that what you meant?